src/shared_utils/shared_utils.star

```diff
@@ -121,3 +121,27 @@ with open("/network-configs/network-configs/bootstrap_nodes.txt") as bootnode_fi
             """,
     )
     return enr_items.output
+
+
+def read_genesis_timestamp_from_config(plan, filename):
+    value = plan.run_python(
+        files={constants.GENESIS_DATA_MOUNTPOINT_ON_CLIENTS: filename},
+        wait=None,
+        run="""
+with open("/network-configs/config.yaml", "r") as f:
+    yaml_content = f.read()
+
+# Manually parse YAML-like content
```

Comment:
WHY? why not use pyyaml?

Reply:
this seems dangerous, lets use some out of the box yaml parser

Reply:
I tried to import yaml, but no packages were found.
wasn't sure how to import a package into run_python

Reply:
fixed